tasks/daemons.js

```diff
@@ -57,7 +57,14 @@ function startDisposableDaemons (callback) {
             throw err
           }
 
-          ipfsNodes[key].setConfig('API', '{"HTTPHeaders": {"Access-Control-Allow-Origin": ["*"]}}', err => {
+          const headers = {
+            HTTPHeaders: {
```

Comment:
Let's put `When ipfs/go-ipfs#1979 is merged we can remove the config settings in the daemon setup` as a TODO here and a issue pointing to it